batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/dao/BookDao.java

```diff
@@ -0,0 +1,8 @@
+package com.lftechnology.batch7crud.dao;
+
+/**
+ * @Author binodnme
+ * Created on 1/17/16
+ */
+public class BookDao {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this empty class, write its code or make it an "interface". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2094)